classMemory.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    "use strict";
+
+    function Memory() {
+        this.num = 0;
+    }
+
+    Memory.prototype.mPlus = function(number) {
```

Comment:
You are creating a memory module here, there is no need for **m** prefix in those methods -- it is implicit.